### PR TITLE
Refs [TASK] [35180] Create socials section

### PR DESCRIPTION
**Pull request by ghost**
https://edu-redmine.sun-asterisk.vn/issues/35180

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: pages/index.pug [hunk 1]
```diff
@@ -23,4 +23,15 @@ section.section__policy
     include home/policy
 section.section__products-showcase
     include home/product_showcase.pug
+<<<<<<< HEAD
```
Comment (manhnt-0960) [inline pages/index.pug:26]: .

File: pages/home/legacy.pug [hunk 2]
```diff
@@ -14,10 +14,16 @@
...
+        +legacy('Home', 'legacy-detail4', './assets/images/product4.jpg')
+        +legacy('OHUI', 'legacy-detail5', './assets/images/product5.jpg')
+        +legacy('DEBORAH LIPPMANN', 'legacy-detail6', './assets/images/product6.jpg')
+        +legacy('HOME', 'legacy-detail7', './assets/images/product7.jpg')
+        +legacy('DEBORA', 'legacy-detail8', './assets/images/product8.jpg')
+
+
+        
+
+
```
Comment (manhnt-0960) [inline pages/home/legacy.pug:29]: xóa dòng trống đi e

Comment (ghost) [reply pages/home/legacy.pug:29]: Dạ vâng ạ

**Pull request closed without merging**